var/spack/repos/builtin/packages/py-flit-core/package.py

```diff
@@ -19,6 +19,7 @@ class PyFlitCore(PythonPackage):
 
     license("BSD-3-Clause")
 
+    version("3.10.1", sha256="66e5b87874a0d6e39691f0e22f09306736b633548670ad3c09ec9db03c5662f7")
```

Comment:
There's actually another new version (since yesterday) :slightly_smiling_face: 

```suggestion
    version("3.11.0", sha256="6ceeee3219e9d2ea282041f3e027c441597b450b33007cb81168e887b6113a8f")
    version("3.10.1", sha256="66e5b87874a0d6e39691f0e22f09306736b633548670ad3c09ec9db03c5662f7")
```

Reply:
I don't need this newer version, but you're welcome to add it in a follow up PR.